services/filesystem-provider/Dockerfile.ci

```diff
@@ -0,0 +1,21 @@
+FROM mcr.microsoft.com/dotnet/sdk:3.1
```

Comment:
Just a note, not requesting change: As per the norm, if the resulting build doesn't work you may need to look at https://github.com/bcgov/mds/blob/develop/openshift4/docker-images/dotnet-31/Dockerfile as that's what we had to use for permissions to get it to work in the past, however that is a redhat image.